src/components/Dao/DaoSchemesPage.tsx

```diff
@@ -80,7 +80,7 @@ class DaoSchemesPage extends React.Component<IProps, null> {
     const allKnownSchemes = [...contributionReward, ...knownSchemes];
 
     const schemeManager = data[1];
-    const schemeManagerActive = getSchemeIsActive(schemeManager);
+    const schemeManagerActive = schemeManager ? getSchemeIsActive(schemeManager) : null;
```

Comment:
schemeManagerActive is active is boolean .. should it get `null` value ?

Reply:
changed to false